runtime/pom.xml

```diff
@@ -22,6 +22,14 @@
       <groupId>org.messaginghub</groupId>
       <artifactId>pooled-jms</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.narayana.jta</groupId>
+      <artifactId>narayana-jta</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss</groupId>
+      <artifactId>jboss-transaction-spi</artifactId>
+    </dependency>
```

Comment:
Is adding narayana actually needed here? Seems strange to be changing overall runtime deps for the extension as a whole in a PR just about adding tests with qpid-jms? (Especially as qpid-jms doesnt do xa, and those tests dont look to do anything with transactions).

This looks like the first change since the last extension release, which I used in JVM and native builds with qpid-jms without these deps present, as part of your PR adding the wrap support to qpid-jms and also mine to make narayana optional in #113 following its previous addition.

Reply:
Nice catch! It seems I added them before and should be removed.

Reply:
I remove them in https://github.com/quarkiverse/quarkus-pooled-jms/pull/122